app/controllers/govspeak_controller.rb

```diff
@@ -0,0 +1,8 @@
+class GovspeakController < ApplicationController
+  def preview
+    skip_authorization
+
+    govspeak_document = Govspeak::Document.new(params["bodyText"])
+    render json: { renderedGovspeak: govspeak_document.to_html }
+  end
+end
```

Comment:
I'm not super familiar with jQuery. Could this render HTML rather than JSON containing HTML and then have the `$.post` request's callback set `$('.govspeak').html(data);` ?